config-bot/config.toml

```diff
@@ -0,0 +1,36 @@
+[git]
+author.name = 'CoreOS Bot'
+author.email = 'coreosbot@fedoraproject.org'
+github.repo.owner = 'coreos'
+github.repo.name = 'fedora-coreos-config'
+github.token.username = 'coreosbot'
+github.token.path = '/var/run/secrets/coreos.fedoraproject.org/github-token/token'
+
+[sync-build-lockfiles]
+builds-base-url = 'https://builds.coreos.fedoraproject.org/prod/streams'
+streams = [
+    'bodhi-updates',
+]
+trigger.mode = 'periodic'
+trigger.period = '15m'
+method = 'push'
+
+[promote-lockfiles]
+source-ref = 'bodhi-updates'
```

Comment:
Should the `source-ref` & `target-ref` be bundled together in an array to support handling multiple stream combinations at the same time or is the idea to run one instance of this tool per lockfile promotion required?

Reply:
Yeah, I think eventually we'll want to support multiple mappings here, e.g. when `branched -> next-devel` is active for example. Punted on this for now. :) But yeah I think we can handle it all in the same instance.